MergeCalendarsTogether.gs

```diff
@@ -84,6 +92,13 @@ function GetStartEndDates () {
   return [startDate, endDate];
 }
 
+function createLogger (){
+  return {
+    info: (...msg)=> { console.log('INFO: ', ...msg) },
+    debug: (...msg)=> { if (VERBOSE_LOGGING) console.log('DEBUG:', ...msg)}
```

Comment:
Added a simple logger function to more easily insert verbose logging as an option without making a mess for everybody.  Anything more than this seemed overkill. Converted all existing console.log invocations to log.info.